src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -124,21 +124,180 @@ class PerformanceLattice(BaseModel):
+        rabiFrequencyErrorRel (Decimal): Total error in the Rabi frequency due to inhomogeneity and
+            variations in time, relative to the specified value. (unitless)
+        rabiFrequencyGlobalErrorRel (Decimal): RMS Rabi frequency variation in time as a relative
+            value. (unitless)
+        rabiFrequencyInhomogeneityRel (Decimal): RMS Rabi frequency inhomogeneity over the user
+            region, relative to the specified value. (unitless)
+        groundDetectionError (Annotated[Decimal, Field(ge=0, le=1)]): Probability of mis-detecting
+            a ground-state atom as a Rydberg-state atom. (unitless)
+        rydbergDetectionError (Annotated[Decimal, Field(ge=0, le=1)]): Probability of mis-detecting
+            a Rydberg-state atom as a ground-state atom. (unitless)
+        groundPrepError (Annotated[Decimal, Field(ge=0, le=1)]): Probability of failing to
+            initialize an atom in the ground state prior to user-programmed coherent evolution, in
+            the absence of any local detuning pattern. Normalized to 1.
+        rydbergPrepErrorBest (Annotated[Decimal, Field(ge=0, le=1)]): Probability of failing to
+            initialize an atom in the Rydberg state by an optimal (for that site) user specified
+            pi-pulse from the ground state at maximum Rabi frequency, in the absence of any local
+            detuning pattern. Normalized to 1.
+        rydbergPrepErrorWorst (Annotated[Decimal, Field(ge=0, le=1)]): Worst-case probability of
+            failing to initialize an atom in the Rydberg state by a user specified pi-pulse from
+            the ground state at maximum Rabi frequency, optimized for a different site, in the
+            absence of any local detuning pattern. Normalized to 1.
+        T1Single (Decimal): Typical lifetime of the Rydberg state for a single non-interacting
+            qubit in the absence of drive, as measured by a pi-wait-pi protocol. (measured in s)
+        T1Ensemble (Decimal): Lifetime of the Rydberg state for an ensemble of non-interacting
+            qubits distributed over the user region, in the absence of drive, as measured by a
+            pi-wait-pi protocol. (measured in s)
+        T2StarSingle (Decimal): Typical dephasing time of a single non-interacting qubit in the
+            absence of drive, as measured by a Ramsey protocol. Includes coherent and incoherent
+            processes. (measured in s)
+        T2StarEnsemble (Decimal): Dephasing time of an ensemble of non-interacting qubits
+            distributed over the user region, in the absence of drive, as measured by a Ramsey
+            protocol. Includes coherent and incoherent processes. (measured in s)
+        T2EchoSingle (Decimal): Typical dephasing time of a single non-interacting qubit in the
+            absence of drive, as measured by a spin-echo dynamical decoupling protocol. This
+            measurement isolates the effects of incoherent processes. (measured in s)
```

Comment:
Does `measured in s` represents `seconds` here?  And can you confirm if the terms like unitless, s, will be elaborated in another PR or you are addressing those now? Other than that looks good.

Reply:
1. Yes, `measured in s` means the quantity has unit seconds.
2. Yes, units like `s`, `(unitless)`, `meters` are address as part of this PR, and there will no other PR
3. Lishuai and I have discussed on the choice of wordings for the units in his CR, and we agree to use these units.

Reply:
Yes. Using "s" instead of "seconds"  was a deliberate decision to be abbreviated, I believe - see https://code.amazon.com/reviews/CR-101330423/revisions/2#/diff

@maolinml I understand that Quera is looking to update some field descriptions in another iteration in the future. For fields like "unitless", I will lean on you as you're close to customer voice to decide if we want to update those fields - if needed, I can publish another PR and update these fields accordingly